coalib/results/TextPosition.py

```diff
@@ -71,11 +70,70 @@ def __ge__(self, other):
         """
         if self.line is None or other.line is None:
             return True
+
         if self.line == other.line:
-            return (True
-                    if self.column is None or
-                    other.column is None
-                    else
-                    self.column >= other.column)
+            if self.column is None or other.column is None:
+                return True
+            else:
+                return self.column >= other.column
         else:
             return self.line > other.line
+
+    def __lt__(self, other):
+        """
+        Test whether ``self`` is behind the other
```

Comment:
`__lt__` actually checks if `self` is *ahead* (or *before*) `other`. Same applies to `__le__`